x-pack/plugins/security/public/authentication/login/login_page.test.tsx

```diff
@@ -183,6 +198,62 @@ describe('LoginPage', () => {
 
       expect(wrapper.find(DisabledLoginForm)).toMatchSnapshot();
     });
+
+    it('renders open in new window button when cookies are disabled and document is embedded inside iframe', async () => {
+      const coreStartMock = coreMock.createStart();
+      httpMock.get.mockResolvedValue(createLoginState());
+
+      Object.defineProperty(window, 'navigator', {
+        value: { cookieEnabled: false },
+        writable: true,
+      });
+      Object.defineProperty(window, 'top', {
+        value: {},
+        writable: true,
+      });
+
+      const wrapper = shallow(
+        <LoginPage
+          http={httpMock}
+          notifications={coreStartMock.notifications}
+          fatalErrors={coreStartMock.fatalErrors}
+          loginAssistanceMessage=""
+        />
+      );
+
+      await act(async () => {
+        await nextTick();
+        wrapper.update();
+      });
+
+      expect(wrapper.find(EuiEmptyPrompt)).toMatchSnapshot();
+    });
+
+    it('renders open in new window button when cookies are disabled and document is not embedded inside iframe', async () => {
```

Comment:
__nit:__ we don't render `open in a new window button` when Kibana isn't embedded, do we?

Reply:
oops :)